openstudiocore/src/utilities/core/UUID.i

```diff
@@ -12,16 +12,13 @@
 namespace openstudio{
 
   #ifdef SWIGRUBY
-    %rename("nil?") isNull();
+    %rename("nil?") is_nil();
```

Comment:
`is_nil` is the native name for the `boost::uuids::uuid` "check if null" method. So I added it just for completeness. We can probably safely remove it.

The rest of the API remains unchanged from the ruby perspective.